src/pipx/venv.py

```diff
@@ -124,6 +140,10 @@ def create_venv(self, venv_args: List[str], pip_args: List[str]) -> None:
         # its contents are additional items (one per line) to be added to sys.path
         pipx_pth.write_text(str(shared_libs.site_packages) + "\n", encoding="utf-8")
 
+        # write venv-specific metadata
+        self.pipx_metadata.venv_args = venv_args
+        self.pipx_metadata.python_version = self.get_python_version()
```

Comment:
nit: comment is misleading since this is updating the metadata, but not actually writing it to a file

Reply:
Just removed that comment, the code explains itself.
Done.